DigitalOffice/Tests/CompanyServiceTests/Commands/AddCompanyCommandTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.CompanyService.Models;
+using LT.DigitalOffice.CompanyService.Commands;
+using LT.DigitalOffice.CompanyService.Database.Entities;
+using LT.DigitalOffice.CompanyService.Mappers.Interfaces;
+using LT.DigitalOffice.CompanyService.Commands.Interfaces;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using Moq;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.CompanyServiceUnitTests.Commands
+{
+    public class AddCompanyCommandTests
+    {
+        private Mock<IValidator<AddCompanyRequest>> validatorMock;
+        private Mock<IMapper<AddCompanyRequest, DbCompany>> mapperMock;
+        private Mock<ICompanyRepository> repositoryMock;
+
+        private IAddCompanyCommand command;
+        private AddCompanyRequest request;
+        private DbCompany createdCompany;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            request = new AddCompanyRequest
+            {
+                Name = "Lanit-Tercom"
+            };
+
+            createdCompany = new DbCompany
+            {
+                Id = Guid.NewGuid(),
+                Name = request.Name,
+                IsActive = true
+            };
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            validatorMock = new Mock<IValidator<AddCompanyRequest>>();
+            mapperMock = new Mock<IMapper<AddCompanyRequest, DbCompany>>();
+            repositoryMock = new Mock<ICompanyRepository>();
+
+            command = new AddCompanyCommand(validatorMock.Object, mapperMock.Object, repositoryMock.Object);
+        }
+
+
+        [Test]
+        public void FailAddCompanyAccordingToValidator()
```

Comment:
Rename tests as we decided on our last conversation. With "should"

Reply:
Fixed.